src/common/options/mds.yaml.in

```diff
@@ -1741,3 +1741,13 @@ options:
   - mds
   flags:
   - runtime
+- name: mds_log_oversegmented_trim_idle_interval
```

Comment:
this config and the related code I don't think we need?

Reply:
Removed.